index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+</head>
```

Comment:
kindly remember that it is very important to declare your CSS file at the head of your page so that your style takes effect on the class, div and tags that you can select in your CSS file.
please add this piece of code after your title and you will see that your `<h1>`  will take the blue color that you assigned to it in your CSS file.
` <link rel="stylesheet" href="style.css">`